.github/workflows/3rd-party-tests.yml

```diff
@@ -0,0 +1,88 @@
+name: Test on 3rd party libraries
+
+on:
+  push:
+    branches:
+      - "master"
+      - "sqlalchemy_pipeline"
+    paths-ignore:
+      - "docs/*"
+      - "tools/*"
+  workflow_dispatch:
+
+concurrency:
+  # Cancel older requests of the same workflow in the same branch.
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  sqlalchemy:
+    # linux should be enough to test if everything works.
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - "3.11"
+        sqlalchemy:
+          - git+https://github.com/sqlalchemy/sqlalchemy.git#egg=SQLAlchemy
+          - sqlalchemy
```

Comment:
This, at the moment, means 1.4.x and will become 2.0.x once the latter is out of beta, right? Maybe it's better to be more explicit?

Reply:
The pip command is using `--pre` so it's installing v2 already. but we can add `>=2`